RPIPs/RPIP-core-dev.md

```diff
@@ -0,0 +1,52 @@
+---
+rpip: TBC 
+title: Protocol Development
+description: Determines the process around core protocol development and funding
+author: Darren Langley (@langers)
+discussions-to: https://dao.rocketpool.net/t/dev-team-funding/2038
+status: Draft
+type: Meta
+created: 2023-10-31
+---
+
+## Abstract
+To ensure the long-term sustainability and continued innovation of the Rocket Pool protocol, this document lays out the high-level roles, responsibilities, processes, and funding for core protocol development.
+
+## Related RPIPs
+- [RPIP-10: pDAO Budget Allocation](https://rpips.rocketpool.net/RPIPs/RPIP-10)
+- [RPIP-24: oDAO Charter](https://rpips.rocketpool.net/RPIPs/RPIP-24)
+- [RPIP-25: RPL Inflation Allocation](https://rpips.rocketpool.net/RPIPs/RPIP-25)
+
+## Specification
+
+### Prioritisation
```

Comment:
I'm unsure if this sufficiently emphasizes the "discussion" aspect. On the last roadmap iteration, the community did some brainstorming that was used as an input into the team's meeting. However, the team's thoughts did _not_ go into the community discussion and there was no iteration. It was a single one-way input, and then the team generated a roadmap.

(this comment will definitely not gate merging of this RPIP as draft, or review)

Reply:
That is fair. 

From our perspective, we were trying to balance making sure the roadmap was community-driven and giving us room to figure out the best release plan/ordering based on technical considerations.

Although, I get your point that the community lost the opportunity to participate further in the prioritisation and inclusion process. The next round is already much more collaborative but we will be mindful of including more iterations.